src/lib/index.js

```diff
@@ -1067,7 +1067,7 @@ function templateFormatString(string, labels, d3locale) {
 
         if(format) {
             var fmt;
-            if(d3locale) {
+            if(d3locale && d3locale.numberFormat) {
```

Comment:
Why do we need this extra condition now?

Reply:
I'll remove this in a future commit. I added it to my TODO above:
> for GL traces, pass through the current d3locale to texttemplateString

Reply:
Fixed in cbbcfcbe3635eb34a51ef2e8a8b35f0c885c7323